src/fiatconnect/saga.ts

```diff
@@ -740,6 +742,24 @@ export function* handleCreateFiatConnectTransfer({
   }
 }
 
+export function* handleKycTryAgain({ payload }: ReturnType<typeof kycTryAgain>) {
+  const { quote, flow } = payload
+
+  try {
+    yield call(deleteKyc, {
+      providerInfo: quote.getProviderInfo(),
+      kycSchema: quote.getKycSchema()!, // it is impossible for kyc schema to be undefined on the quote
+    })
+
+    navigate(Screens.KycLanding, { quote, flow, step: 'one' })
```

Comment:
separate from this PR but it seems like the KYC steps could be named more descriptively